### DISCUSSION TITLE
linux demo运行时报段错误，带文字的pag文件就会报错

### DISCUSSION THREAD

**Discussion by wfd2022** [Tencent/libpag#2143] [Q&A] [answered]
在linux编译可成功通过，运行Linux的demo时报段错误，测试发现用pag::PAGFile::Load函数加载带文字的pag文件就会报错，没有文字的可以成功运行
![企业微信截图_17083056192668](https://github.com/Tencent/libpag/assets/91302457/919e77d7-1703-4f82-b771-60de85943c1d)

**Comment by wfd2022** [comment, marked as answer]
切换版本后可正常运行，可能是最新版本的bug

**Comment by kevingpqi123** [comment]
建议先查看下 pag 文件的大小，如果小于 1 KB，基本上是文件没有正确拉取下来
pag 的文件配置了 lfs，如果没有执行 git lfs pull，默认文件是无效的

**Reply by wfd2022** [reply]
pag文件是有效的，可以预览，我用其他版本没有出现段错误
